buildSrc/src/main/kotlin/DependencyConfig.kt

```diff
@@ -48,6 +48,9 @@ fun Project.configureDependencies() {
         maven("https://jitpack.io") {
             name = "JitPack"
         }
+        maven("https://nexuslite.gcnt.net/repos/other/") {
```

Comment:
Is folialib published to maven central, or only here?

Reply:
Only here